tests/Teams/ReplyToChannel/Bots/ReplyToChannel.cs

```diff
@@ -27,7 +27,7 @@ protected override async Task OnMessageActivityAsync(ITurnContext<IMessageActivi
         private async Task SendMessageAndLogActivityId(ITurnContext turnContext, string text, CancellationToken cancellationToken)
         {
             var replyActivity = MessageFactory.Text(text);
-            replyActivity.ApplyConversationReference(turnContext.Activity.GetConversationReference());
+            //replyActivity.ApplyConversationReference(turnContext.Activity.GetConversationReference());
```

Comment:
Is this necessary?

Reply:
Yes, it's uncommented. 

Since this isn't a reply to an existing thread you still need to apply the conversation reference.